static/css/main.css

```diff
@@ -190,7 +187,10 @@ header.post-title {
 .footer {
     font-size: 10pt;
     text-align: center;
-    margin-top: 10px;
+    /* This is used to make the footer "flush" with the rest of the page */
+    margin-top: auto;
+    padding-top: 10px;
+    padding-bottom: 8px;
```

Comment:
The same argument applies here as at https://github.com/swicg/activitypub.rocks/pull/40/files#r2325296852.